app/scenes/Document/components/EditableTitle.tsx

```diff
@@ -128,14 +152,28 @@ const EditableTitle = React.forwardRef(
         onBlur={onBlur}
         placeholder={placeholder}
         value={value}
-        $emojiWidth={emojiWidth}
         $isStarred={document.isStarred}
+        $containsEmoji={!!document.emoji}
         autoFocus={!document.title}
         maxLength={DocumentValidation.maxTitleLength}
         readOnly={readOnly}
         dir="auto"
         ref={ref}
       >
+        <EmojiPicker
+          disclosure={
+            <EmojiButton size={32} onClick={(ev) => ev.stopPropagation()}>
+              {document.emoji ? (
+                <Emoji size="24px" native={document.emoji} />
```

Comment:
Probably should be the same size?

Reply:
With and without `size="24px"` respectively :point_down: 

![Screenshot from 2022-11-03 12-30-59](https://user-images.githubusercontent.com/15694566/199663283-a27b84ac-bb15-4c15-bfd1-1351968008ef.png)
![Screenshot from 2022-11-03 12-30-14](https://user-images.githubusercontent.com/15694566/199663340-65cda924-9ea4-4e4f-85d9-cbd6d7512760.png)

(The one without occupies `36px` width)